azure-pipelines.yml

```diff
@@ -0,0 +1,90 @@
+parameters:
+- name: module_names
+  type: object
+  default:
+  - nidigital
+  - nitclk  
+  - nifgen
+  - nidcpower
+  - nidmm
+  - niscope
+  - nimodinst
+  - nise
+  - niswitch
+
+# A pipeline with no CI trigger
+trigger: none
+
+# specific path
+pr:
+  branches:
+    include:
+    - master
+    - releases/*
+  paths:
+    exclude:
+    - CHANGELOG.md
```

Comment:
I don't know if this works, but if you can exclude all markdown docs `*.md` it will be more robust.

Reply:
Support for wildcards was added just this week: https://docs.microsoft.com/en-us/azure/devops/release-notes/2021/sprint-192-update#support-for-wild-cards-in-path-filters.

Reply:
Will try this out.

Reply:
Will try this with next change.

Reply:
did you try out? or did you mean next PR?